python/setup.py

```diff
@@ -25,14 +25,26 @@
 ]
 sklearn = ["scikit-learn"]
 torch = ["torch>=1.8.1", "torchvision>=0.9.1"]
+tf = ["tensorflow>=2.5.0"]
 jupyter = ["matplotlib", "jupyterlab"]
 aws = ["boto3", "botocore"]
 gcp = ["gcsfs"]
 docs = ["sphinx"]
 video = ["ffmpeg-python", "scenedetect"]
 youtube = ["pafy", "youtube_dl"]
 mlflow = ["mlflow==1.20.2"]
-all = sklearn + torch + jupyter + gcp + docs + video + youtube + mlflow + aws
+all = (
```

Comment:
Should we discourage users from installing all? Feels like it's really heavy to install both frameworks and also both GCP and aws support etc

Reply:
Yea, it is good call. I am open to it. This is really way to much to install.